tests/s25Main/mapGenerator/testHeadQuarters.cpp

```diff
@@ -202,4 +248,26 @@ BOOST_AUTO_TEST_CASE(PlaceHeadQuarters_places_hqs_for_any_player_number_on_suita
     }
 }
 
+BOOST_AUTO_TEST_CASE(PlaceHeadQuarters_without_suitable_position_throws_exception)
+{
+    MapExtent size(32, 32);
+    RandomUtility rnd(0);
+
+    for(int players = 1; players < 8; players++)
```

Comment:
Instead of this (which makes the test run much longer) use `rttr::test::randomNumber(1, 8)`, see the testMap.cpp (IIRC)
The semantic is a bit different: Using the loop means that for the number of players there might be different handling and all must be tested while the random number means "any number of player will trigger the behavior that should be tested" (and still, over time, tests all cases) while cutting the runtime to 1/8 :)